oceanprefilter/prefilter_test.go

```diff
@@ -0,0 +1,119 @@
+package oceanprefilter
+
+import (
+	"fmt"
+	"image"
+	"math"
+	"os"
+	"path/filepath"
+	"testing"
+	"image/jpeg"
+	xgb "github.com/Elvenson/xgboost-go"
+	"github.com/Elvenson/xgboost-go/activation"
+)
+
+
+func TestGeneralModel(t * testing.T) {
```

Comment:
use a better name for this test to describe what it is doing -- could be called TestXGBoostInference